scripts/publish

```diff
@@ -15,3 +15,8 @@ do
     twine upload dist/*
     popd
 done
+
+# Publish Titiler meta package
+rm -rf dist
+python setup.py sdist
+twine upload dist/*
```

Comment:
Note you could also create a `titiler/metapackage` folder to have the metapackage "as a module" instead of at the top level.

Reply:
yeah, might be confusing 🤷